notebooks/20220913-fraction-mtx-after-subtracting-mgx.ipynb

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
<ul><li>I am really suprised at how high the <strong>mean_f_n_hashes </strong>is ....am i understanding this correctly, that in several cases &gt;90% of unique hashes in the transcriptome are not in metagenome? it looks like abundance weighting resolves some of this, which you are attributing to sequencing errors, etc. but that would be a super high error rate right?</li><li>Why did you choose k = 31? How do these fractions change with more stringest k = 51? </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
1. Ya this is fairly un-intuitive, but remember that for every single sequencing error, we get 31 k-mers that are erroneous and will probably only occur in a data set once (think of a sliding window over the error, where the error occurs in every position during that sliding window). This means that distinct k-mers that represent errors really blow up -- abundance-weighting them gets rid of a lot of these problems. 
2. I chose 31 because it's an ok default :) in reality, we could do it with all k-mer sizes, and i'm continuing the downstream taxonomic profiling with all k-mer sizes. The numbers decrease by ~3% for k=21, and increase by 3% for k = 51. These changes could be due to errors tho ... errors will have less of an impact at smaller k-sizes and bigger impact at bigger ksizes. Also, the conversation we had about the euk rna db (now recorded in #5), I only have that database for k = 31.